src/Powercord/plugins/pc-moduleManager/components/manage/Base.jsx

```diff
@@ -79,6 +80,48 @@ class Base extends React.Component {
     return null;
   }
 
+  renderItems (items) {
+    const [ PC, noPC ] = this._partition(items, p => p.entityID.startsWith('pc-'));
+    return <>
+      {this.renderPCCategory(PC)}
+      {noPC.map(item => this.renderItem(item))}
+    </>;
+  }
+
+  renderPCCategory (items) {
+    return (
+      <Category
+        name={Messages.POWERCORD_PLUGINS_CORE}
+        description={Messages.POWERCORD_SETTINGS_ADVANCED_DESC}
+        opened={this.state.PCPluginsCategoryOpen}
+        onChange={() =>
+          this.setState({
+            PCPluginsCategoryOpen: !this.state.PCPluginsCategoryOpen
+          })
+        }
+      >
+        {this.renderEnableAllButton(items)}
+        {items.map(item => this.renderItem(item))}
+      </Category>
+    );
+  }
+
+  renderEnableAllButton (items) {
+    const needEnable = items.filter(p => !p.ready);
+    return (
+      <ButtonItem
+        button={(needEnable.length) ? Messages.POWERCORD_PLUGINS_ENABLE_ALL_PC_GO : Messages.POWERCORD_PLUGINS_ALL_ENABLED}
```

Comment:
You could just make a simple button without a label in front, and put the label in it. I don't think it's even worth checking if everything is enabled and disable it if it's the case (avoids having to iterate over the plugin array).

Reply:
well, if forced enable does not cause problems...